data/index.php

```diff
@@ -50,7 +50,6 @@ function get_title ($page) {
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta itemprop="image" content="https://valadoc.org/images/preview.png">
```

Comment:
This is a valid meta line. It's a schema.org meta for the website image. http://schema.org/WebSite